cuda_core/tests/test_module.py

```diff
@@ -13,23 +13,23 @@
 
 from cuda.core.experimental import ObjectCode, Program, ProgramOptions, system
 
+code = """
```

Comment:
Minor style suggestion: `SAXPY_KERNEL`

`code` was fine as a variable name in function scope, but at module scope I'm usually seeing names capitalized and with a meaningful (longer) name. Reads better.